### DISCUSSION TITLE
New RPM community venue

### DISCUSSION THREAD

**Discussion by pmatilai** [rpm-software-management/rpm#2013] [Announcements]
As of today, we're opening up the [Discussions](https://github.com/rpm-software-management/rpm/discussions) forum as a new venue for community interaction.

Why, you ask, when we have all these mailing lists? The sad fact is that the mailing lists are all but dead, to the point that even us maintainers miss the rare post on them, leading them even more dead because few people like talking to themselves. Yet, clearly there is a need for a place to ask questions and discuss various aspects of rpm and its future, and based on evidence people are more inclined to file a ticket to do this rather than post on a mailing list. That, or remain silent. Neither is a particularly good outcome.

We hate the potential vendor lock-in as much as anybody, so these discussions will always be mirrored to [rpm-maint mailing list ](http://lists.rpm.org/pipermail/rpm-maint/)along with the ticket and PR notifications. Other than that, we'll see how it goes.

**Comment by Conan-Kudo** [comment]
Also note that we do have Matrix rooms for real-time communication:

RPM: https://matrix.to/#/#rpm:matrix.org

RPM-Ecosystem: https://matrix.to/#/#rpm-ecosystem:matrix.org
